app/core/build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
     testImplementation "com.nhaarman:mockito-kotlin:${versions.mockitoKotlin}"
     testImplementation "org.jdom:jdom2:2.0.6"
     testImplementation "org.koin:koin-test:${versions.koin}"
+    implementation 'com.android.support:mediarouter-v7:27.1.1'
```

Comment:
This seems unrelated to the change. Please try to keep pull requests as minimal as possible.